testSuite/scripts/test_upload_block_blob.py

```diff
@@ -559,14 +559,14 @@ def test_download_blob_exclude_flag(self):
         except:
             self.fail('error parsing the output in JSON Format')
         # Number of expected successful transfer should be 18 since two files in directory are set to exclude
-        self.assertEquals(x.TransfersCompleted, 17)
+        self.assertEquals(x.TransfersCompleted, 14)
```

Comment:
Ditto, why the change in behaviour with so little change to the test?

Reply:
Explained in above comment.